### ISSUE TITLE
Ensure "Edit Site" option is consistent in both places (top toolbar and floating toolbar)

### ISSUE THREAD

**Issue by annezazu** [Automattic/wp-calypso#94079]
#### Steps to reproduce

1. Look at the front end of a logged in site with a block theme
2. Notice there's "edit site" in the floating toolbar and "edit site" in the top toolbar. 
3. Click each and notice they take you different places. Edit site in floating toolbar takes you to the Site Editor > Templates where as "Edit site" in the top toolbar will take you to the respective template you are choosing.

#### What I expected

Expect the buttons to act the same. 

#### What happened instead

The buttons took me two different places. 

#### Browser / OS version

MacOS, Chrome

#### Screenshot / Video

https://github.com/user-attachments/assets/e8a947c8-1184-49a2-bd82-5454d71e36de

**Comment by ramonjd** [comment]
I noticed there's an item from @richtabor to remove the "Edit site" link from the floating action toolbar completely.

It might save adding the logic in Calypso to take you to the respective template you are choosing.

Then again, while testing it, I noticed that `canvas=edit` is missing from the link.

https://thingsireadonthetoilet.wordpress.com/wp-admin/site-editor.php?postType=wp_template&postId=pub%2Fassembler%2F%2Fhome

vs

https://thingsireadonthetoilet.wordpress.com/wp-admin/site-editor.php?postType=wp_template&postId=pub%2Fassembler%2F%2Fhome&canvas=edit

This is a DotCom backend change regardless of which happens. 👍🏻

**Comment by ramonjd** [comment]
> This is a DotCom backend change regardless of which happens. 

If we remove the "Edit site" link, I'd assume the "Customize" link should stay for classic themes?

<img width="203" alt="Screenshot 2024-09-09 at 9 26 40 AM" src="https://github.com/user-attachments/assets/c1e4a5b6-150e-4195-8458-09d8a8be60c1">

**Comment by ramonjd** [comment]
I've appended "canvas=edit" to these links so that the editor opens.


https://github.com/user-attachments/assets/afb0068a-62b5-4522-8542-fdfa30530357


D162881-code

**Comment by ramonjd** [comment]
D162881-code now removes the Edit Site from the toolbar completely

**Comment by niranjan-uma-shankar** [comment]
Related issue:

https://github.com/Automattic/wp-calypso/issues/94358

**Comment by ramonjd** [comment]
I think this PR is waiting on a redesign. See @mmtr's [comments](https://github.com/Automattic/wp-calypso/issues/94358#issuecomment-2373830979):

> That action bar with just an ellipsis button looks quite wrong to me, so I'm moving this back to "Needs design" because I think we need to revisit how the action bar should look like under these circumstances.

D162881-code removes the "Edit Site" link and the result is a lonely ellipsis button. 😄

**Comment by richtabor** [comment]
> I noticed there's an item from @richtabor to remove the "Edit site" link from the floating action toolbar completely.

Yes, let's just remove "Edit site" from the action bar. It's not the same as "Customize" — which is a much more contained environment where it's harder to break your website: https://github.com/Automattic/wp-calypso/issues/95363

**Comment by simison** [comment]
@richtabor are we removing "edit post" link as well? It's in the masterbar, too:

![Image](https://github.com/user-attachments/assets/2b0fab87-e1c0-4820-8376-a4c6c68d2d34)

![Image](https://github.com/user-attachments/assets/e93d3d1a-cf6a-4dcb-b490-65baa24a203a)

**Comment by ramonjd** [comment]
Progress on this has stalled. Just x-linking pfYzsZ-On-p2#comment-966

**Comment by simison** [comment]
@ramonjd question of removing entirely shouldn't block you from improving/removing edit-links meanwhile, especially if it seems like Action bar will remain there. :-)

**Comment by ramonjd** [comment]
> question of removing entirely shouldn't block you from improving/removing edit-links meanwhile

Thanks for the clarification @simison 👍🏻

**Comment by richtabor** [comment]
Yes, can we remove "Edit Site" at the very least?

**Comment by ramonjd** [comment]
For folks coming back to this, D162881-code revision ID removed the "Edit Site" option.

**Comment by simison** [comment]
I noticed we also have "Customize" visible in the masterbar alongside "Edit site" :-(

<img width="555" alt="Image" src="https://github.com/user-attachments/assets/9b781bdd-6732-4a9a-b3dc-474c2e1b5642" />

Filed an issue:
- https://github.com/Automattic/wp-calypso/issues/97735

**Comment by ramonjd** [comment]
Let's get rid of all `Edit` links in the toolbar.

Also, the "Get theme:" for logged in users looking at their own site. There's no need to get the theme that's already installed.

The window prompt to copy is also from the jurassic period.

Maybe something like this?

https://github.com/user-attachments/assets/817b748c-5114-4416-b7bd-4a0046a2278e

**Comment by ramonjd** [comment]
> Yes, can we remove "Edit Site" at the very least?

I ported the phab patch that removed the "Edit Site" link over to 169373-gh-Automattic/wpcom

**Comment by davemart-in** [comment]
> My vote is to remove the floating toolbar completely.

I'd ask that we please not do this. It is still valuable, especially for blogger engagement.

Happy to:

A) Cull links that no longer make sense.
B) Make the action bar optional - so anyone could turn it off if they want (but still defaulting it to be on)
C) Redesign the action bar to align more with Gutenberg. Here were a few designs I shared:

![Image](https://github.com/user-attachments/assets/baf075a2-a034-40d0-aefc-72690fd48ff1)

**Comment by ramonjd** [comment]
> I'd ask that we please not do this. It is still valuable, especially for blogger engagement.

👍🏻 

>  Redesign the action bar to align more with Gutenberg. Here were a few designs I shared:

Looks great, thanks for sharing. 

I'll leave 169373-ghe-Automattic/wpcom as is (just removing Edit Site) for now so that's addressed at least for this issue. 

The redesign is a chunkier bit of work we can swing back to.

**Comment by ramonjd** [comment]
169373-ghe-Automattic/wpcom has been committed.

Can we close this?

**Comment by ramonjd** [comment]
Closing for now as "Edit Site" no longer exists in the floating toolbar.

See also: https://github.com/Automattic/jetpack/pull/41137
